testdata/ecosystem/pipelineruns/kn-apply-multiarch.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  name: kn-apply-pz-run
+spec:
+  pipelineSpec:
+    tasks:
+    - name: run-kn-apply
+      taskRef:
+        name: kn-apply
+        kind: ClusterTask
```

Comment:
Cluster tasks are gone in 1.17, that is why you are creating new tests for ecosystem. Update it to use resolver tasks.

Reply:
Done